src/pages/docs_sdk_purchase-subscription.html

```diff
@@ -67,6 +67,24 @@ <h3 id="notes">Notes</h3>
     </li>
   </ul>
 
+  <h3 id="advanced">Advanced</h3>
+
+  <ul>
+    <li>You can support calculating tax at checkout. To enable automatic tax:
+      <ul>
+        <li>Add a coupon to your Stripe account under <a href="https://dashboard.stripe.com/coupons" target="_blank">Products > Coupons</a></li>
+        <li>Then you need to set tax behavior on a product</li>
+      </ul>
+    </li>
+    <li>You can allow promo codes at checkout. To enable promo codes:
+      <ul>
+        <li>You need to activate tax collection under <a href="https://dashboard.stripe.com/settings/tax/activate" target="_blank">Settings > Tax Settings</a></li>
+        <li>Set an origin address via this link and configure tax settings</li>
+        <li>Then add a promotion code for that coupon</li>
+      </ul>
```

Comment:
Something like:

```
<li>Finally, visit your app's Payments Portal in your <a href="https://v1.userbase.com/" target="_blank">Admin panel</a>, click "Show Advanced", and enable the option to "Allow promotion codes in Checkout"</li>
```

for both this and tax. Whatever you think feels right :)

Reply:
@j-berman Done!